src/components/Confirm/index.js

```diff
@@ -22,11 +22,11 @@ const FileViewer = ({file:{preview, type}}) =>
   </object>
 
 
-const Capture = ({capture:{file}}) =>
+const Capture = ({capture:{image}}) =>
   <div className={style.captures}>
-    {isOfFileType(['pdf'], file) ?
-      <FileViewer file={file}/> :
-      <img src={file.preview} className={style.image} />
+    {isOfFileType(['pdf'], image) ?
+      <FileViewer file={image}/> :
+      <img src={URL.createObjectURL(image)} className={style.image} />
```

Comment:
how heavy is `createObjectURL`? Should this value not be cached? Can it be added to the `Blob` object?